contents.json

```diff
@@ -5687,6 +5687,11 @@
         "category": "uicollectionview",
         "description": "UICollectionViewCell with checkbox when it isSelected and empty circle when not - like Photos.app 'Select' mode.",
         "homepage": "https://github.com/yonat/CheckmarkCollectionViewCell"
+    }, {
+        "title": "BatteryView",
+        "category": "ui",
+        "description": "Simple battery shaped UIView",
```

Comment:
@yonat  could you end the description with `.`?

Reply:
Sure, done.